xml/System.Runtime.InteropServices.Swift/SwiftIndirectResult.xml

```diff
@@ -24,8 +24,14 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents the Swift return buffer context.</summary>
+    <remarks>
+      <para> This struct is used to access the return buffer when interoping with Swift functions that return non-frozen structs.
+ It provides a pointer to the memory location where the result should be stored.
+ </para>
+      <para> Here's an example of how a SwiftIndirectResult can be declared:
+ <code lang="csharp"> [UnmanagedCallConv(CallConvs = [typeof(CallConvSwift)])] [LibraryImport("SwiftLibrary", EntryPoint = "export")] public static extern void swiftFunction(SwiftIndirectResult result); </code></para>
```

Comment:
With this code be pretty-printed? Having it as a single line makes for poor documentation.

Reply:
Here's the preview, it looks icky: https://review.learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.swift.swiftindirectresult?view=net-9.0&branch=pr-en-us-10154#remarks

Let me find out how to make it look better.

Reply:
Fixed. Let's wait for the preview now.

Reply:
@AaronRobinsonMSFT looks nice now! https://review.learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.swift.swiftindirectresult?view=net-9.0&branch=pr-en-us-10154#examples

![image](https://github.com/user-attachments/assets/615d4122-3bf6-42cd-ad2a-f24cb564096d)